test/mocha/10-restrictions.js

```diff
@@ -88,6 +88,55 @@ describe('restrictions', function() {
       err.message.should.equal('Duplicate restriction.');
     });
 
+  it('should throw ValidationError with no duration', async function() {
```

Comment:
I think an additional test that demonstrates that test what is/is not a `DuplicateError` would help to ensure that the index is functioning as expected.  Restrictions with the same `resource`, `zone` with distinct `methodOptions.foo` values should now be allowed where they were not before.

Reply:
Added new test `should not throw DuplicateError if duration is different`.